### PR TITLE
Place an expand/collapse button at each license document

### PR DESCRIPTION
**Pull request by rockwillj**
Some license documents, such as Apache License, are too long to scroll them.

I added the style to place an expand/collapse button at the top right of each license document.

I implemented the feature with pure CSS because I don't want to call `webView.getSettings().setJavaScriptEnabled(true)`.

### Limitation

- The feature does not work in Android 4.2.2 because it does not support general sibling selectors (`A ~ B`)
  - It is not critical issue, I think,  because Android 4.2.x is old enough and at least you can see header of the license document.

### CODE REVIEW

File: example/src/main/assets/licenses.html [hunk 1]
```diff
@@ -43,14 +43,44 @@
...
+    }
   </style>
 </head>
 <body>
     <div class="library">
       <!-- https://opensource.org/licenses/Apache-2.0 -->
-      <h1 class="title">Android Orma</h1>
-      <p class="notice">Copyright (c) 2015 FUJI Goro (gfx)<br/>SQLite.g4 is: Copyright (c) 2014 by Bart Kiers<br/></p>
-      <p><a href="https://github.com/ReactiveX/RxJava">https://github.com/ReactiveX/RxJava</a></p>
+      <h1 class="title">Android Support Libraries</h1>
```
Comment (rockwillj) [inline example/src/main/assets/licenses.html]: I don't know why this change is included.
I just executed `./gradlew generateLicensePage` to generate `license.html`.
`licenses.html` was not updated to the latest state?

Comment (uzzu) [reply example/src/main/assets/licenses.html]: I tried to generate license page on master, it seems to be old, so I will approve.

Comment (SabiRini) [reply example/src/main/assets/licenses.html]: Yes it was old. Same here #45

**Comment by uzzu** [comment]
@rockwillj 
Approve this feature. Thanks for contribute 👍 
Please resolve conflict, and then I will merge this PR.

**Comment by rockwillj** [comment]
@uzzu I resolved conflict. Thank you for approval 😃

**Review by uzzu — APPROVED** [review]
